Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -1065,24 +773,22 @@ public override Attribute MakeAttribute (Color fore, Color back)
 
 		public override void Suspend ()
 		{
-			if (reportableMouseEvents.HasFlag (Curses.Event.ReportMousePosition))
-				StopReportingMouseMoves ();
+			StopReportingMouseMoves ();
 			Platform.Suspend ();
 			Curses.Window.Standard.redrawwin ();
 			Curses.refresh ();
-			if (reportableMouseEvents.HasFlag (Curses.Event.ReportMousePosition))
-				StartReportingMouseMoves ();
+			StartReportingMouseMoves ();
 		}
 
 		public override void StartReportingMouseMoves ()
 		{
-			Console.Out.Write ("\x1b[?1003h");
+			Console.Out.Write ("\x1b[?1003h\x1b[?1015h\x1b[?1006h");
```

Comment:
It would make this all more readable and maintainable if we had constants defined for these sequences.

e.g.

```cs
Console.Out.Write (CSI_AllMouseTracking + CSI_EnableurxvtMouseMode + CSI_EnableSGRMouseMode);
```

Reply:
Done. Thanks.